.travis.yml

```diff
@@ -0,0 +1,141 @@
+# Based on the "trust" template v0.1.2
+# https://github.com/japaric/trust/tree/v0.1.2
+
+dist: trusty
+language: rust
+
+addons:
+  apt:
+    packages:
+      - libsqlite3-dev
+
+services: 
+  - docker
+  - mysql
+  - postgres
+  - sqlite3
+
+sudo: required
+
+env:
+  global:
+    - CRATE_NAME=sql-composer
+    - SQLITE_DATABASE_URL=/tmp/sql_composer_test.db
+    - SQLITE_MEMORY_DATABASE_URL=:memory
+    - PG_DATABASE_URL=postgres://postgres@localhost/
+    - PG_EXAMPLE_DATABASE_URL=postgres://postgres@localhost/sql_composer_example
+    - MYSQL_DATABASE_URL=mysql://travis@localhost/sql_composer_test
+    - MYSQL_EXAMPLE_DATABASE_URL=mysql://travis@localhost/sql_composer_example
+    - MYSQL_UNIT_TEST_DATABASE_URL=mysql://travis@localhost/sql_composer_unit_test
+
+before_script:
+  - mysql -e "create database sql_composer_test;grant all on \`sql_composer_%\`.* to 'travis'@'%';" -uroot
+
+matrix:
+  include:
+    # Android
+    - env: TARGET=aarch64-linux-android DISABLE_TESTS=1
+    - env: TARGET=arm-linux-androideabi DISABLE_TESTS=1
+    - env: TARGET=armv7-linux-androideabi DISABLE_TESTS=1
+    - env: TARGET=i686-linux-android DISABLE_TESTS=1
+    - env: TARGET=x86_64-linux-android DISABLE_TESTS=1
+
+    # iOS
+    - env: TARGET=aarch64-apple-ios DISABLE_TESTS=1
+      os: osx
+    - env: TARGET=armv7-apple-ios DISABLE_TESTS=1
+      os: osx
+    - env: TARGET=armv7s-apple-ios DISABLE_TESTS=1
+      os: osx
+    - env: TARGET=i386-apple-ios DISABLE_TESTS=1
+      os: osx
+    - env: TARGET=x86_64-apple-ios DISABLE_TESTS=1
+      os: osx
+
```

Comment:
We can cut down on the number of included entries in the matrix while getting a working travis build.  Especially these ones with DISABLE_TESTS=1.